.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -10,6 +10,11 @@ assignees: ''
 **Describe the bug**
 A clear and concise description of what the bug is.
 
+**App versions**
+Android:
+
+Web:
```

Comment:
Same comment as on the other related PR - there's no version in the web app atm. Even if there were, the user probably wouldn't know with which version of the web app was used to create the survey.

Reply:
Removed. I'll update https://github.com/google/ground-platform/issues/2015 as well